### ISSUE TITLE
Engines render as… pink

### ISSUE THREAD

**Issue by rspeed** [bigorangemachine/ksp-kronalutils#33]
Version: 0.0.4 - 1.1 Pre  
KSP Version: 1.1.2  
OS: OS X 10.11.4  
GPU: GeForce 8600M GT

In screenshots, all engines are pink. I tried various settings, but was unable to find a workaround. Additionally, I had to disable all of the graphics options other than FSAA, otherwise the entire screenshot was pink. I suspect those issues may be related.

![front_methane converter_1](https://cloud.githubusercontent.com/assets/520459/15033118/a6086eea-1235-11e6-9dce-653a3d5e60ef.png)

**Comment by CobaltWolf** [comment]
One of my contributors (who relies on KVV to take orthographics for my manual) ran into the same issue with 1.1.2 using whichever version is on Spacedock at time of writing. He is going to try again with the 1.1-pre release.

**Comment by rspeed** [comment]
Yeah, I'm already using that version.

**Comment by bigorangemachine** [comment]
I have opened a thread on the KSP forums asking for help regarding this issue. I think it works if you don't use OpenGL. 

The issue is how the original "edge detection" shaders were recompiled is no longer supported in unity5

**Comment by rspeed** [comment]
I'm on OS X, so it's the only option.

**Comment by CobaltWolf** [comment]
Yeah my manual editor uses a mac. Everything worked fine on my end. I think we'll just have me take the screenshots then.

**Comment by rspeed** [comment]
In the meantime I can borrow a computer from a family member.

I played around with using some tools to bring ships into Blender, with moderate success – and only because I write Python for a living. But ithe tools were just too buggy to be reliable. The renders looked fantastic, though.
